### ISSUE TITLE
ERROR - Read timed out

### ISSUE THREAD

**Issue by SamWalkerRxSense** [Wisser/Jailer#25]
Jailer 9.1.4.3

I have a large sql file, about 450MB, trying to use the "Import Sql Data" tool.
After about 7 minutes into the load I get a bunch of "WARN   - cancellation request received" and 
then I get "ERROR  - "Read timed out" in statement".
later in the log I get this.
Caused by: com.microsoft.sqlserver.jdbc.SQLServerException: Read timed out

This error happens on a remote SQL Server instance.
I ran the same process locally and it works fine, no error.

I assume it's just taking too long and there is some time out setting, but I couldn't find anything about it in the documentation.

Thanks.

**Comment by Wisser** [comment]
Do you also get the error if the file is executed with another tool (MS SQL Developer etc.)?
It might help to reduce the number of "rows per statements" when exporting.
Or you increase the timeout. This should be possible with a queryTimeout/lockTimeout parameter in the JDBC URL. See https://docs.microsoft.com/en-us/sql/connect/jdbc/setting-the-connection-properties?view=sql-server-ver15

**Comment by SamWalkerRxSense** [comment]
Thanks for the quick reply.

The export "rows per statement" was 50.
Also, on the import, parallel threads = 50.

I just tried with export "rows per statement" = 10
Import with 50 threads, and it worked,  Finished without error in about 10 minutes.

I've tried Sql Server Management Studio and sqlcmd, but their too slow.  Those take hours, single thread I assume, so not practical.

I can use bcp which is super fast (only a couple minutes) but I prefer the upsert SQL Jailer generates.  It's takes less thinking ... usually.
